lib/perl/Genome/Config/Parser/YAML.pm

```diff
@@ -19,4 +20,11 @@ sub parse {
     return LoadFile($filename);
 }
 
-1;
\ No newline at end of file
+sub write {
+    my ($self, $filename, $data) = validate_pos(@_, 1, 1, 1);
+    die('Failed to provide filename!') unless $filename;
+    die('Failed to provide content!') unless $data;
```

Comment:
Using `validate_pos` I can definitely remove lines 25-26 and line 16. Do you suggest to also consolidate the remaining tests in the `parse` subroutine using `validate` instead of `validate_pos`?